services/authors/src/errors-handling/convert-err-to-http-form.js

```diff
@@ -0,0 +1,9 @@
+import { errorMessages } from '../config.js';
+
+export const convertErrToHttpForm = (err) => {
+  const responseStatus = err.status ? err.status : 500;
+  const message =
+    responseStatus === 500 ? errorMessages.INTERNAL_SERVER_ERROR : err.message;
```

Comment:
I suggest moving the response statuses into a separate object with constants, for example:
```
responseStatusCode = {
BAD_REQUEST = 400,
NOT_FOUND = 404
}
```
etc.